06_gpu_and_ml/embeddings/text_embeddings_inference.py

```diff
@@ -1,7 +1,8 @@
 # ---
 # cmd: ["modal", "run", "06_gpu_and_ml/embeddings/text_embeddings_inference.py::embed_dataset"]
+# tags: ["use-case-embedding-model", "featured"]
 # ---
-# # Run TextEmbeddingsInference (TEI) on Modal
+# # Run an embedding model with TextEmbeddingsInference (TEI) on Modal
```

Comment:
Could we separate this into a different PR? Would rather keep PRs single purpose when possible, especially small ones like this.